src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,39 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.dto.CartDTO;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/carts")
 public class CartController {
+
+
+    @PostMapping
+    public ResponseEntity<Void> createCart(@RequestBody CartDTO cartDto){
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping
+    public ResponseEntity<List<CartDTO>> getItemsFromCart(){
+        return ResponseEntity.ok(null);
+    }
+
+    @PutMapping(value = "{itemId}")
+    public ResponseEntity<CartDTO> addItemToCart(@PathVariable Long itemId){
```

Comment:
No dobra, mamy itemId, a do ktorego koszyka go dodac? Czy nie powinien to byc url /v1/carts/{cartId}?productId=blablabla ?

Reply:
Alternatywnie mogloby byc updateCart(CartDto cart)